website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: Progressive Delivery using Flagger
+hide_title: true
+sidebar_position: 6
+---
+
+import TierLabel from "../_components/TierLabel";
+
+# Progressive Delivery using Flagger <TierLabel tiers="enterprise" />
+
+Weave GitOps Enterprise includes the ability to view [Flagger](https://docs.flagger.app/) [canary](https://docs.flagger.app/usage/how-it-works#canary-resource) objects information in the UI through the Applications > Delivery view.
+
+This guide uses Flux manifests to install Flagger and Linkerd. Using Flux allows us to manage our cluster applications in a declarative way through changes in a Git repository.
+
+## Installing Flagger using Flux
+
+For the Flagger installation, a Kustomization file will be used.
```

Comment:
> A Kustomization file will be used

Why? Can we explain this to the reader. 

I think we will also need to show a sample repository structure or at least talk about an assumed path which is already being reconciled with Flux.

Reply:
In the case of Linkerd, the use of a `secretGenerator` makes it very easy to read files (certificates) and turn them into secrets.